parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
The other `cse_` scripts specified `bash`, and this script was using a comparison that isn't defined in POSIX `sh`.